DefInjected/InspirationDef/Inspirations.xml

```diff
@@ -4,71 +4,71 @@
   <!-- EN: go frenzy -->
   <Frenzy_Go.label>szalony chód</Frenzy_Go.label>
   <!-- EN: Inspired: Go frenzy -->
-  <Frenzy_Go.baseInspectLine>zainspirowany(a): Szalony chód</Frenzy_Go.baseInspectLine>
+  <Frenzy_Go.baseInspectLine>Inspiracja: Szalony chód</Frenzy_Go.baseInspectLine>
   <!-- EN: [PAWN_nameIndef] has experienced an inspiration!\n\n[PAWN_pronoun] will move faster for the next 8 days. -->
-  <Frenzy_Go.beginLetter>{0} doznał/a inspiracji. [PAWN_pronoun] będzie chodzić szybciej przez następne 8 dni.</Frenzy_Go.beginLetter>
+  <Frenzy_Go.beginLetter>[PAWN_nameIndef] doświadczył{0_gender ? : a} inspiracji!\n\n[PAWN_pronoun] będzie chodzić szybciej przez następne 8 dni.</Frenzy_Go.beginLetter>
   <Frenzy_Go.beginLetterLabel>Inspiracja: szalony chód</Frenzy_Go.beginLetterLabel>
   <!-- EN: {0} is no longer inspired (go frenzy). -->
-  <Frenzy_Go.endMessage>{0} nie jest już zainspirowany(a) (szalony chód).</Frenzy_Go.endMessage>
+  <Frenzy_Go.endMessage>{0} nie jest już zainspirowan{0_gender ? y : a} (szalony chód).</Frenzy_Go.endMessage>
   
   <!-- EN: shoot frenzy -->
   <Frenzy_Shoot.label>szał strzelania</Frenzy_Shoot.label>
   <!-- EN: Inspired: Shoot frenzy -->
-  <Frenzy_Shoot.baseInspectLine>zainspirowany(a): Szał strzelania</Frenzy_Shoot.baseInspectLine>
+  <Frenzy_Shoot.baseInspectLine>Inspiracja: Szał strzelania</Frenzy_Shoot.baseInspectLine>
   <!-- EN: [PAWN_nameIndef] has experienced an inspiration!\n\n[PAWN_pronoun] will shoot more accurately for the next 8 days. -->
-  <Frenzy_Shoot.beginLetter>{0} jest zainspirowany(a). [PAWN_possessive] będzie strzelać celniej przez następne 8 dni.</Frenzy_Shoot.beginLetter>
+  <Frenzy_Shoot.beginLetter>[PAWN_nameIndef] doświadczył{0_gender ? : a} inspiracji!\n\n [PAWN_possessive] będzie strzelać celniej przez następne 8 dni.</Frenzy_Shoot.beginLetter>
```

Comment:
```suggestion
  <Frenzy_Shoot.beginLetter>[PAWN_nameIndef] doświadczył{PAWN_gender ? : a} inspiracji!\n\n [PAWN_pronoun] będzie strzelać celniej przez następne 8 dni.</Frenzy_Shoot.beginLetter>
```

[PAWN_pronoun] - on, ona, ono
[PAWN_objective] - go, ją,to
[PAWN_possessive] - jego, jej, tego